tracee/tracee_test.go

```diff
@@ -120,16 +120,40 @@ func TestReadArgFromBuff(t *testing.T) {
 			},
 			expectedArg: map[string]string(map[string]string{"sa_family": "AF_INET", "sin_addr": "255.255.255.255", "sin_port": "65535"}),
 		},
+		{
+			name: "sockAddrT - AF_UNIX",
+			input: []byte{12, //sockAddrT
+				0,    // Dummy tag
+				1, 0, //sa_family=AF_UNIX
+				47, 116, 109, 112, 47, 115, 111, 99, 107, 101, 116, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 101, 110, 0, 0, 0, // sun_path=/tmp/socket
```

Comment:
the max length of `sun_path` is 108 bytes. here you give 114 bytes, which explains the extra 6 bytes that fails the test.

Reply:
Ah nice catch - you're right.